Feature material

lennart-bader opened the pull request:
Habe mal ein paar Design-Änderungen vorgenommen.
Der geschlossene PR ist hier auch drin enthalten (Wischen im Terminal).
Ansonsten nicht so viel neues. Lebt hier eigentlich noch jemand ;-)

<img src="https://cloud.githubusercontent.com/assets/4501333/13707597/b8ee34aa-e7aa-11e5-871d-9b41c281bba5.png" width="250" />
<img src="https://cloud.githubusercontent.com/assets/4501333/13707604/bf2eee72-e7aa-11e5-9a4e-d880d82286b4.png" width="250" />

Commit:
Changed to Material Design: New Navigation Drawer

Commit:
Changed Terminal Create Topic to use ViewPager
Fixed invisible text in TerminalDetail

Commit:
Fixed wrong text color for settings-switch
Changed Toolbar color

lennart-bader commented:
@rleh 
Weißt du was zu Travis? Der meckert rum :smile:

rleh commented:
Ja:

```
* What went wrong:
A problem occurred configuring project ':app'.
> failed to find Build Tools revision 23.0.2
```

Also scheinbar sind die Build-Tools in `gradle.build` nicht gleich denen in `.travis.yml`.

rleh commented on app/build.gradle:
```diff
@@ -3,7 +3,7 @@ apply plugin: 'com.google.gms.google-services'
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    buildToolsVersion "23.0.2"
```
HIer liegt das Problem.
Bitte hier die Änderungen zurücknehmen oder in `.travis.yml` ebenfalls entsprechend ändern.

rleh commented:
Edit: Scheinbar sind die Build-Tools in `gradle.build` nicht gleich denen in `.travis.yml`.

rleh commented on build.gradle:
```diff
@@ -5,7 +5,7 @@ buildscript {
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:2.1.0-alpha1'
```
Gibt es einen speziellen Grund hierfür?

rleh committed:
reverted buildtool/etc. version updates

rleh commented:
Travis builded gleich wieder, ich merge dann mal :+1:

rleh committed:
update buildtool version (gradle AND travis), update dependencies, include only gcm from google play services (prevents OOM in travis)

rleh committed:
travis: add android component 'tools'

(see https://github.com/travis-ci/travis-ci/issues/5036#issuecomment-153297292)

rleh commented:
Ok, Travis stirbt nun an `OOM` :unamused: 

Dann machen wir jetzt ein Update auf Buildtools Version 23.0.2; Das [gestaltet sich als nicht so einfach](https://github.com/travis-ci/travis-ci/issues/5036#issuecomment-153297292)

Außerdem wird nur noch `gcm` aus den _Google Play Services_ eingebunden, das sorgt für geringeren Speicherbedarf beim builden.

lennart-bader committed:
Changed colors

lennart-bader commented:
<img src="https://cloud.githubusercontent.com/assets/4501333/14218346/dbd4a76a-f853-11e5-856d-891416a2798b.png" width="250" style="float: left;" />

<img src="https://cloud.githubusercontent.com/assets/4501333/14218364/f361ce94-f853-11e5-8db1-d0636061efb8.png" width="250" style="float: left;" />

rleh commented:
Ich finde das Design so ok.

Soll ich morgen?

lennart-bader commented:
Von mir aus ja.
Wobei die Tabs z.B. noch zu überarbeiten sind. Zählt meiner Meinung nach aber eigentlich zum Issue für die Toolbar.
